cutplace/ranges.py

```diff
@@ -281,7 +282,7 @@ def __init__(self, description, default=None):
                             # Handle "...".
                             raise errors.InterfaceError(
                                 'ellipsis (...) must be preceded and/or succeeded by number', location)
-                        else:
+                        else:  # maybe useless?!
```

Comment:
Replace whole `else` clause with `assert False` and test what happens with a range of `'1 x 2'`.